modelforge/potential/models.py

```diff
@@ -609,27 +791,7 @@ def prepare_inputs(self, data: NNPInput, only_unique_pairs: bool = True):
 
         return nnp_input
 
-    def _set_dtype(self):
```

Comment:
The `self._dtype` variable isn't used in any of the models or in the class itself, so it became redundant.

This is a result of defining each tensors created in  `__init__`  either as `nn.Parameter` or `nn.Buffer`, so that we can handle the move to a different `dtype` or `device` with a simple `model.to(device, dtype)`. That approach is much more explicit and makes it clear what the used `dtype` (or `device) is.